Luis.FuzzyMatching/Solution/FuzzyMatching.Core/RuntimeClient.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.CognitiveServices.Language.LUIS.Utilities.FuzzyMatching.Core.Helpers;
+using Microsoft.Azure.CognitiveServices.Language.LUIS.Utilities.FuzzyMatching.Core.Helpers.MatrixOperations;
+using Microsoft.Azure.CognitiveServices.Language.LUIS.Utilities.FuzzyMatching.Core.Services.TFIDFControllers;
+using Microsoft.Azure.CognitiveServices.Language.LUIS.Utilities.FuzzyMatching.Definitions.Configs;
+using Microsoft.Azure.CognitiveServices.Language.LUIS.Utilities.FuzzyMatching.Definitions.Enums;
+using Microsoft.Azure.CognitiveServices.Language.LUIS.Utilities.FuzzyMatching.Definitions.Models;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.CognitiveServices.Language.LUIS.Utilities.FuzzyMatching.Core
+{
+    public class RuntimeClient
+    {
+        public static readonly float MatchingThreshold = Constants.MatchingThreshold;
+
+        public List<MatchResult> MatchEntities(ProcessedDataset processedDataset, List<string> dataset, string testSentence, MatchingMethod matchingMethod, float threshold = default, int ngramSize = default)
```

Comment:
a little bit nitpicky, but would it be better to use 'inputSentence' or 'utterance' instead of testSentence?